mease_lab_to_nwb/convert_ced/cedstimulusinterface.py

```diff
@@ -31,13 +30,16 @@ def check_module(nwbfile, name, description=None):
         return nwbfile.create_processing_module(name, description)
 
 
-def intervals_from_traces(traces):
+def intervals_from_traces(recording: RecordingExtractor):
     """Extract interval times from TTL pulses."""
+    traces = recording.get_traces(channel_ids=[1, 2])
```

Comment:
@CodyCBakerPhD why `channel_ids` is always `[1, 2]`? I would add `channel_ids` as an extra arg